src/configuration/app/build.md

```diff
@@ -163,3 +163,32 @@ hooks:
         fi
         drush -y updatedb
 ```
+
+## How can I use my custom environment variables?
+
+Custom environment variables can be defined in the console. They will be available through the $PLATFORM_VARIABLES but to extract them, you need a couple of extra steps. 
```

Comment:
Custom environment variables can be defined using the [management console](/administration/web/configure-project.md#variables) or with the [Platform.sh CLI](/development/variables.md). They will be available within your during build as the base64-encoded JSON `PLATFORM_VARIABLES`. 

As an example, you can unpack `PLATFORM_VARIABLES` with a tool like `jq` in your build hook to access them:

```yaml
```

Reply:
Err... I like your text better than mine. How do I accept it?

Reply:
Bad reviewing here on my part. Second sentence should be:

> They will be available during build as the base64-encoded JSON `PLATFORM_VARIABLES`.

Reply:
As far as I know use what you like of my edit, push it, then I approve the changes. Not sure if there's an "accept changes".